src/lib/Display.ts

```diff
@@ -27,16 +37,32 @@ export interface Display {
 	 * Output table to stdout only if the "--json" flag is not present.
 	 */
 	displayTable<R extends Ux.Table.Data>(data: R[], columns: Ux.Table.Columns<R>): void;
+
+	spinnerStart(msg: string, status?: string): void;
+
+	spinnerUpdate(status: string): void;
+
+	spinnerStop(status: string): void;
 }
 
 export class UxDisplay implements Display {
 	private readonly displayable: Displayable;
+	private readonly spinner: Spinner;
 
-	public constructor(displayable: Displayable) {
+	public constructor(displayable: Displayable, spinner: Spinner) {
 		this.displayable = displayable;
+		this.spinner = spinner;
+	}
+
+	public displayWarning(msg: string): void {
+		this.displayable.warn(msg);
 	}
 
 	public displayInfo(message: string): void {
+		this.displayable.info(message);
+	}
+
+	public displayLog(message: string): void {
 		this.displayable.log(message);
 	}
```

Comment:
Can you help me understand the difference here?

Reply:
The different methods style their output differently. See the following picture.
![Screenshot 2024-06-26 at 12 09 12 PM](https://github.com/forcedotcom/sfdx-scanner/assets/4054488/537bad3f-ee0a-4800-a3ac-2d00c0750f76)

Reply:
Update, I actually figured out how to log an error without exiting, and it looks like this:
![Screenshot 2024-06-26 at 12 47 35 PM](https://github.com/forcedotcom/sfdx-scanner/assets/4054488/815192aa-b585-4c98-a991-fc3797b8e470)
I'll integrate this shortly.